contracts/lib/arbitration/IArbitrable.sol

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
We already have these interfaces in the Court repo. Could we import them from there? Or have wherever a single source of truth?

Reply:
The source of truth should be aragonOS, although we cannot do that cause we are using different solidity versions :/